src/main/java/me/mrCookieSlime/Slimefun/hooks/github/ContributionsConnector.java

```diff
@@ -53,13 +69,16 @@ public String getURLSuffix() {
 	private void computeContributors(JsonArray array) {
 	    for (int i = 0; i < array.size(); i++) {
 	    	JsonObject object = array.get(i).getAsJsonObject();
-	    	
+
 	    	String name = object.get("login").getAsString();
 	    	int commits = object.get("contributions").getAsInt();
 	    	String profile = object.get("html_url").getAsString();
-	    	
-	    	if (!blacklist.contains(name)) {
-	    		Contributor contributor = SlimefunPlugin.getUtilities().contributors.computeIfAbsent(name, key -> new Contributor(name, profile));
+
+	    	if (nameFormat.matcher(name).matches() && !blacklist.contains(name)) {
+	    		Contributor contributor = SlimefunPlugin.getUtilities().contributors.computeIfAbsent(
+	    				name,
```

Comment:
If we now start mapping multiple gh accounts to a single mc account we should probably use the alias as the key too.

Reply:
Nevermind, I think the double keys is fine. We have the (MC: x) suffix now.